config/api.yml

```diff
@@ -2336,7 +2336,17 @@
         - ems_infra_show_list
         - ems_block_storage_show_list
       - :name: create
-        :identifier: ems_infra_new
+        :identifiers:
+        - :klass: EmsInfra
+          :identifier: ems_infra_new
+        - :klass: EmsCloud
+          :identifier: ems_cloud_new
+        - :klass: Provider #EmsContainer
+          :identifier: ems_container_new
+        - :klass: EmsNetwork
+          :identifier: ems_network_new
+        - :klass: Provider
+          identifier: provider_foreman_add_provider
```

Comment:
missing the leading `:` here

Reply:
That's fine, since this relies on the aliases that you aren't a fan of anyway I doubt this is getting in ever.